packaging/debian/watch

```diff
@@ -0,0 +1,3 @@
+version=3
+opts=filenamemangle=s/.+\/(\d\S*)\.tar\.gz/fatrace-$1\.tar\.gz/ \
```

Comment:
What does this line mean/do?

Reply:
That's a standard screenscraping recipe for GitHub's /release pages to scan for a tarball to download. See https://manpages.debian.org/buster/devscripts/uscan.1.en.html . However, copy&paste fail, obviously I need to replace the "fatrace" with "sscg". D'oh!

Reply:
Fixed, `uscan --report` is ok now:
```
uscan: Newest version of sscg on remote site is 2.6.2, local version is 0
uscan:  => Newer package available from:
        => https://github.com/sgallagher/sscg/archive/refs/tags/sscg-2.6.2.tar.gz
```
On the real Debian upload, debian/changelog will obviously have the current version, the 0 is just a placeholder for not having to update it all the time.